manim/mobject/coordinate_systems.py

```diff
@@ -212,6 +213,85 @@ def add_coordinates(self, x_vals=None, y_vals=None):
         self.add(self.get_coordinate_labels(x_vals, y_vals))
         return self
 
+    def get_line_graph(
+        self,
+        x_values: Iterable[float],
+        y_values: Iterable[float],
+        z_values: Optional[Iterable[float]] = None,
+        line_color: Colors = YELLOW,
+        add_vertex_dots: bool = True,
+        vertex_dot_radius: float = DEFAULT_DOT_RADIUS,
+        vertex_dot_style: Optional[dict] = None,
+        **kwargs
+    ) -> VDict:
+        """Draws a line graph connecting the vertices formed from zipping
+        x_values, y_values and z_values. Also adds :class:`~.Dot`s at the
+        vertices if add_vertex_dots is set to True.
+
+        Parameters
+        ----------
+        x_values : :class:`Iterable`
+            Iterable of values along the x-axis.
+        y_values : :class:`Iterable`
+            Iterable of values along the y-axis.
+        z_values : :class:`Iterable`, optional
+            Iterable of values (zeros if z_values is None) along the z-axis.
+        line_color : :class:`Colors`
+            Color for the line graph.
+        add_vertex_dots : :class:`bool`
+            Whether or not to add :class:`~.Dot` at each vertex.
+        vertex_dot_radius : :class:`float`
+            Radius for the :class:`~.Dot` at each vertex.
+        vertex_dot_style : :class:`dict`
+            Style arguments to be passed into :class:`~.Dot` at each vertex.
+        kwargs : Any
+            Additional arguments to be passed into :class:`~.VMobject`.
```

Comment:
Typings don't have to be added in both docstring and method header.

```suggestion
        x_values
            Iterable of values along the x-axis.
        y_values
            Iterable of values along the y-axis.
        z_values
            Iterable of values (zeros if z_values is None) along the z-axis.
        line_color
            Color for the line graph.
        add_vertex_dots
            Whether or not to add :class:`~.Dot` at each vertex.
        vertex_dot_radius
            Radius for the :class:`~.Dot` at each vertex.
        vertex_dot_style
            Style arguments to be passed into :class:`~.Dot` at each vertex.
        kwargs
            Additional arguments to be passed into :class:`~.VMobject`.
```